packages/site/cms-site/src/blocks/DamVideoBlock.tsx

```diff
@@ -1,30 +1,70 @@
 "use client";
 import * as React from "react";
-import styled from "styled-components";
+import styled, { css } from "styled-components";
 
 import { DamVideoBlockData } from "../blocks.generated";
 import { withPreview } from "../iframebridge/withPreview";
 import { PreviewSkeleton } from "../previewskeleton/PreviewSkeleton";
+import { VideoPreviewImage, VideoPreviewImageProps } from "./helpers/VideoPreviewImage";
 import { PropsWithData } from "./PropsWithData";
 
+interface DamVideoBlockProps extends PropsWithData<DamVideoBlockData> {
+    aspectRatio?: string;
+    previewImageSizes?: string;
+    renderPreviewImage?: (props: VideoPreviewImageProps) => React.ReactElement;
+}
+
 export const DamVideoBlock = withPreview(
-    ({ data: { damFile, autoplay, loop, showControls } }: PropsWithData<DamVideoBlockData>): JSX.Element => {
+    ({
+        data: { damFile, autoplay, loop, showControls, previewImage },
+        aspectRatio = "16x9",
+        previewImageSizes,
+        renderPreviewImage,
+    }: DamVideoBlockProps) => {
         if (damFile === undefined) {
             return <PreviewSkeleton type="media" hasContent={false} />;
         }
 
+        const [showPreviewImage, setShowPreviewImage] = React.useState(true);
+        const hasPreviewImage = previewImage && previewImage.damFile;
```

Comment:
I don't think `damFile` is a boolean, right? 
Currently the variable `hasPreviewImage` would be the value of `damFile` instead of `true`. 

```suggestion
        const hasPreviewImage = Boolean(previewImage && previewImage.damFile);
```

Reply:
did catched that for the youtube block, but missed it here, thanks 👍